pkgs/shells/nushell/plugins/default.nix

```
@@ -1,10 +1,21 @@
-{ lib, newScope, IOKit, CoreFoundation, Foundation, Security }:
-
-lib.makeScope newScope (self: with self; {
-  gstat = callPackage ./gstat.nix { inherit Security; };
-  formats = callPackage ./formats.nix { inherit IOKit Foundation; };
-  polars = callPackage ./polars.nix { inherit IOKit Foundation; };
-  query = callPackage ./query.nix { inherit IOKit CoreFoundation; };
-  regex = throw "`nu_plugin_regex` is no longer compatible with the current Nushell release.";
-  net = callPackage ./net.nix { inherit IOKit CoreFoundation; };
-})
+{
+  lib,
+  newScope,
+  IOKit,
+  CoreFoundation,
+  Foundation,
+  Security,
+  dbus,
+}:
+lib.makeScope newScope (
+  self: with self; {
```

```suggestion
  self: {
```
this probably avoids the infinite recursion

---

That worked, had to add `callPackage` as well.